content/Visibility/Visibility.facts.md

```diff
@@ -0,0 +1,56 @@
+### Facts for Visibility lesson:
+
+Visibility defines the accessibility of a property and a method of a certain class. 
+
+Visibility keywords are defined as the following:
+
+ - `public` sets the accessibility of a property or method of a class everywhere.
+
+ - `private` sets the accessibility of a property or method only within a class itself.
+
+ - `protected` sets the accessibility of a property or method only within a class itself and its child or parent classes.
+
+Class properties and methods can be declared as `public`, `private`, and `protected`. Declaring a method without a visibility keyword is implicitly defined as `public`.
+
+Code:
+
+```php
+<?php
+class Person
+{
+    public $name = "Anna";
+    private $age;
+
+    public function setAge($newAge)
+    {
+        if ($this->checkAge($newAge)) {
+            $this->age = $newAge;
+        }
+    }
+
+    public function getAge()
+    {
+        return $this->age;
+    }
+
+    private function checkAge($age)
+    {
+        if ($age > 0) {
+            return true;
+        }
+        return false;
+    }
+}
```

Comment:
I like that accessors are being introduced to the student but I think it's also important to stress the importance of using them. In general, accessors should be used and class attributes should be private or protected as a best practice. Should this be discussed in a separate chapter or can it be added somewhere in this chapter?